integration_tests/dbt_project.yml

```diff
@@ -1,5 +1,5 @@
 name: 'salesforce_integration_tests'
-version: '1.0.2'
+version: '1.1.0'
```

Comment:
I know this wasn't part of your update, but in order for your `consistency_columns` validation to properly work across both the source and the transform package, we will want to change the scope of the `+schema:` models config. Otherwise, when I run this locally I see numerous failures because my staging models are not being materialized in the `*_dev` schema.

Therefore, we will need to change lines 7-11 to be the following:
```yml
models:
  +materialized: table
  +schema: "salesforce_{{ var('directed_schema','dev') }}"
```

Reply:
For example, without this change I see 304 row failures which is not expected 😱 

![image](https://github.com/user-attachments/assets/3036db50-8862-450b-b27c-0893ebd0a3f2)